test/lib/ex_aws/kms/integration_test.exs

```diff
@@ -0,0 +1,107 @@
+defmodule ExAws.KMSIntegratinTest do
+  use ExUnit.Case, async: true
+
+  defp key_id do
+    System.get_env("AWS_KEY_ARN")
+  end
```

Comment:
So, I'm slightly wary of having this test in. Is this an environment variable that is normally set?